### DISCUSSION TITLE
Pole Figures issue

### DISCUSSION THREAD

**Discussion by King-wang-H** [mtex-toolbox/mtex#1088] [Ask Anything]
Hi, everyone.I am new in using MTEX to analyze the microstructure of magnesium alloys.
Now I am doing texture analysis of rolled magnesium alloy, and I have a problem in the process of treating rolled magnesium alloy with MTEX. It is that when a texture analysis is performed on an EBSD data obtained from a RD(rolling direction)-ND(normal direction) plane is performed, the Pole Figures made by MTEX and made by Channenl 5 has a big difference (the graphs produced by the two software are shown below), I would like to ask what is the reason? Is it true that MTEX can only make pole diagrams of RD(rolling direction)-TD (transverse direction) planes?
Hope to have experienced people to help solve the confusion by a proper way, thank you in advance.
![Pole Figure(1)](https://user-images.githubusercontent.com/85743677/121698267-69c6b200-cb00-11eb-8a96-77b7f143c4bb.jpg)

Wang

**Comment by lfgmorales** [comment]
Dear King-Wang-H



When you import your cpr / ctf file, are you using the appropriate choice of reference frame on mtex import_wizard, to have a 1 to 1 comparison between the measured map and the map you will work with on MTEX?



cheers



Luiz





Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________
From: King-wang-H ***@***.***>
Sent: 11 June 2021 16:01:01
To: mtex-toolbox/mtex
Cc: Subscribed
Subject: [mtex-toolbox/mtex] Pole Figures issue (#1088)


Hi, everyone.I am new in using MTEX to analyze the microstructure of magnesium alloys.
Now I am doing texture analysis of rolled magnesium alloy, and I have a problem in the process of treating rolled magnesium alloy with MTEX. It is that when a texture analysis is performed on an EBSD data obtained from a RD(rolling direction)-ND(normal direction) plane is performed, the Pole Figures made by MTEX and made by Channenl 5 has a big difference (the graphs produced by the two software are shown below), I would like to ask what is the reason? Is it true that MTEX can only make pole diagrams of RD(rolling direction)-TD (transverse direction) planes?
Hope to have experienced people to help solve the confusion by a proper way, thank you in advance.
[Pole Figure(1)]<https://user-images.githubusercontent.com/85743677/121698267-69c6b200-cb00-11eb-8a96-77b7f143c4bb.jpg>

Wang

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1088>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZ5L5QQ4YW7REYMKJ3TSIJJ3ANCNFSM46QZTP3A>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1088", "url": "https://github.com/mtex-toolbox/mtex/discussions/1088", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by King-wang-H** [reply]
Hello, thank you for your reply.When importing EBSD data, I carefully thought about the problem of reference frame. Below are the two IPF plotting I made. However, when I continued to make pole figure on this basis, different texture states appeared.
![1](https://user-images.githubusercontent.com/85743677/121702598-9977b900-cb04-11eb-90ac-c88f2b5d3b3b.jpg)
[
![2](https://user-images.githubusercontent.com/85743677/121702621-a1375d80-cb04-11eb-8a6a-59d8b2dca40e.jpg)
](url)
But when I used the EBSD data from RD-TD plane to draw the pole figure, I found that the pole figure made by MTEX and Channel5 was indeed the same.So I wonder if MTEX can't make pole figure of RD-ND plane.Can you help me to answer it?

**Reply by King-wang-H** [reply]
Dear Luiz

Thank you again for your reply. and the following is the code I used to draw the pole figure. Is it correct?
Or do you have any other code to draw a pole figure?[
![3](https://user-images.githubusercontent.com/85743677/121704505-68988380-cb06-11eb-86de-33d7cee68a3e.png)
](url)
Looking forward to your reply!

**Comment by lfgmorales** [comment]
Dear King-Wang-H,

To my understanding, RD, TD and RD are just names you give to your external reference frame, and I am not entirely sure I understood yoi correctl when you say that the plots are the same when you plot the pole figures of your ebsd data measured on RD-TD?

Also, why do you have plotx2north on your code, after impoeting your data with x to the east? Can you try to plot without that and see if the pole figures make sense?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 11 Jun 2021, at 16:41, King-wang-H ***@***.***> wrote:

﻿

Dear Luiz
Thank you again for your reply. The following is the code I used to draw the pole figure. Is it correct?Or do you have any other code to draw a pole figure?

[3]
<url>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1088#discussioncomment-858526>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6IUVQPK7UPTE5WSJ3TSIOBRANCNFSM46QZTP3A>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1088#discussioncomment-858526", "url": "https://github.com/mtex-toolbox/mtex/discussions/1088#discussioncomment-858526", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by King-wang-H** [reply]
Dear Luiz

The RD, ND and TD I mentioned here refer to the three directions of the magnesium strip in the rolling process (as shown in the figure below). The RD-TD plane and RD-ND plane are planes composed of these three directions, and also refer to the observation scanning surface when obtaining EBSD data.
[
![4](https://user-images.githubusercontent.com/85743677/121767610-c327f300-cb8b-11eb-9cf3-452e85613c24.png)
](url)
When I used MTEX and Channel 5 to make the pole figure of RD-ND plane, the texture distribution was completely different without any similar features. But, the same texture distribution characteristics are presented when the pole figure of the RD-TD plane is made (as shown in the figure below).
[
![5](https://user-images.githubusercontent.com/85743677/121767612-c9b66a80-cb8b-11eb-978b-7c1e38decbde.png)
](url)
After I removed the code plotx2north and ran it, it only changed the X axis and Y axis directions of the pole figure, without changing the special configuration of the texture.

Therefore, my biggest puzzle is why the texture distribution characteristics on the pole figure made by MTEX and Channel5 with the same EBSD data are completely different or even have no similar characteristics when doing RD-TD plane?

Cheers

King-wang-H
